addons/controllers/calico/calicoconfig_controller.go

```diff
@@ -46,3 +88,181 @@ func (r *CalicoConfigReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		For(&cniv1alpha1.CalicoConfig{}).
 		Complete(r)
 }
+
+// ReconcileCalicoConfig reconciles CalicoConfig data values secrets
+func (r *CalicoConfigReconciler) ReconcileCalicoConfig(
+	ctx context.Context,
+	cluster *clusterapiv1beta1.Cluster,
+	calicoConfig *cniv1alpha1.CalicoConfig) (_ ctrl.Result, retErr error) {
+
+	var patchCRD bool
```

Comment:
question: `patchCRD` seems a little bit odd, I am assuming you meant `patchCR` or `patchConfig`

Reply:
changed to ```patchCalicoConfig```